knotx-core/src/main/java/com/cognifide/knotx/util/DataObjectsUtil.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Knot.x - Reactive microservice assembler - API
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.util;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Objects;
+import io.vertx.rxjava.core.MultiMap;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class DataObjectsUtil {
+
+  /**
+   * Equality operator for Multimap objects
+   *
+   * @param self - compared Multimap object self
+   * @param that - compared Multimap object that
+   * @return - true if objects are equals, false otherwise
+   */
+  public static boolean equalsMultimap(MultiMap self, MultiMap that) {
+    return Objects.equal(self.names(), that.names()) &&
+        self.names().stream().allMatch(name -> that.contains(name) && self.getAll(name).equals(that.getAll(name)));
```

Comment:
Multimap is vert.x interface so please add comment here:
MultiMap represents String keys to a List of String values.